frontend/src/__tests__/cypress/cypress/support/commands/odh.ts

```diff
@@ -315,6 +328,147 @@ declare global {
         response: OdhResponse<RegisteredModelList | undefined>,
       ): Cypress.Chainable<null>;
 
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/:pipelineId/versions/:pipelineVersionId`,
+        options: { path: { namespace: string; pipelineId: string; pipelineVersionId: string } },
+        response: OdhResponse<PipelineVersionKFv2>,
```

Comment:
I believe that every pipeline endpoint has the ability to also return a `GoogleRpcStatusKF`.
Confirm with @Gkrumbach07 

```suggestion
        response: OdhResponse<PipelineVersionKFv2 | GoogleRpcStatusKF>,
```

Reply:
@christianvogt thanks I will confirm this with Gage,but  as I  looked at  `PipelineVersionKFv2` type  we do have `error` parameter which is of type   `GoogleRpcStatusKF`

Reply:
that is correct: see this api for the response types: https://www.kubeflow.org/docs/components/pipelines/v2/reference/api/kubeflow-pipeline-api-spec/#operation--apis-v2beta1-runs--run_id--get